pkg/runner-provisioner/provisioner.go

```diff
@@ -70,6 +65,13 @@ func (p *RunnerProvisioner) ProvisionRunner(ctx context.Context, runnerName stri
 		return err
 	}
 
+	p.logger.Infof("creating runner with name %s", runnerName)
+	jitConfig, err := p.createRunner(ctx, runnerName)
+	if err != nil {
+		return err
+	}
+	p.logger.Infof("created runner with name %s", runnerName)
```

Comment:
Do we need both logger statements? The first one seems sufficient. This is a nit tho

Reply:
These are some of the reasons:
1. To be consistent with the VM deploy above
2. To make sure that the current creation does not block
3. Easier tracking in a big log file. It is easy to track if you have one runner at a time. But once you start havign multiple logs for different runners, it becomes hard to track